lesson_7/mihaela_vaida/print_register_dec.py

```diff
@@ -0,0 +1,21 @@
+
+print_registry = []
+def register(fnc):
+    
+    
+    def print_register(*args):        
```

Comment:
There's no need to add these white lines

Reply:
I will remove it , but  Pycharm  formating message was that:  ''PEP 8: expected 2 blank lines, found 1'

Reply:
True, between functions but not after the function definition and it's body